src/test/java/PoseBufferTest.java

```diff
@@ -1,4 +1,5 @@
-import edu.wpi.first.math.geometry.Pose2d;
+//TODO comment this back in
+/*import edu.wpi.first.math.geometry.Pose2d;
```

Comment:
Comment this back in and upgrade your wpilib if you haven't already, should work fine